scripts/src/chartrepomanager/chartrepomanager.py

```diff
@@ -246,6 +246,11 @@ def update_chart_annotation(category, organization, chart_file_name, chart, repo
         vendor_name = out["vendor"]["name"]
         annotations["charts.openshift.io/provider"] = vendor_name
 
+    if "charts.openshift.io/certifiedOpenShiftVersions" in annotations:
+        full_version = annotations["charts.openshift.io/certifiedOpenShiftVersions"]
+        ver = semver.VersionInfo.parse(full_version)
+        annotations["charts.openshift.io/certifiedOpenShiftVersions"] = ".".join((str(ver.major), str(ver.minor)))
```

Comment:
IMHO, `annotations["charts.openshift.io/certifiedOpenShiftVersions"] = f"{ver.major}.{ver.minor}"` is more readable, i.e. the intention is clearer.

Reply:
Done. 908e8605354d7c73c1171c77f37b9ffeee336197